README.md

```diff
@@ -212,16 +211,9 @@ PUT /circles/{id}
     "vision": {string},
     "mission": {string},
     "aim": {string},
-    "expectationsForMembers": {expectationsForMembers},
-    "members": [
-        {
-            "id": {userId},
-            "email": {email},
-            "invitationState": {invitationState:invited|member}
-        }
-    ],
```

Comment:
It would be nice to save the knowledge that this is how we think of adding members and writing expectations, for now.

Actually, I think `expectationsForMembers` should be set on creation, and so should an initial member list. That isn't included currently.